### DISCUSSION TITLE
Suffix for csv writer

### DISCUSSION THREAD

**Discussion by kaelabruin** [bonsai-rx/bonsai#2015] [Q&A]
Hi, 

Is it possible to use the suffix format %03d.csv for the end of my filepath in csvwriter? I want the files to be counted as 000, 001, 002, ect.

Thank you,

Kai

**Comment by glopesdev** [comment]
@kaelabruin this is actually a great suggestion to resolve a long outstanding issue on how to format filename suffixes. I will move this into a separate issue as a proposal and if we can gather enough support maybe we can get it into the next release.

**Reply by glopesdev** [reply]
For reference, this is the existing issue: https://github.com/bonsai-rx/bonsai/issues/1294
